mautic/mautic
# Minify HTML email

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This PR added  new option to enable minify HTML output during the sending. It should increase deliverability of email 




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Go to Config   and Email Settings and enable Minify HTML

![image](https://user-images.githubusercontent.com/462477/218007822-299c9633-414a-45bf-a34c-74a224e80be5.png)

3. Send broadcast email or campaign email
4. Check email source If was minified. 
5. Check If all tokens was replaced properly
